internal/broker/testdata/TestIsAuthenticated/golden/authenticating_with_password_refreshes_expired_token/first_call

```diff
@@ -1,11 +1,3 @@
 access: granted
-data: |-
-    {"userinfo": {
-    		"name": "test-user@email.com",
-    		"uuid": "test-user-id",
-    		"gecos": "test-user",
-    		"dir": "/home/test-user@email.com",
-    		"shell": "/usr/bin/bash",
-    		"groups": [ {"name": "remote-group", "ugid": "12345"}, {"name": "linux-local-group", "ugid": ""} ]
-    }}
+data: '{"userinfo":{"name":"test-user@email.com","uuid":"test-user-id","dir":"/home/test-user@email.com","shell":"/usr/bin/bash","gecos":"test-user","groups":[{"name":"remote-group","ugid":"12345"},{"name":"linux-local-group","ugid":""}]}}'
```

Comment:
I think you get a better formatting when generating the golden files and formatting the data there to have ident, and easier to read, as it was before.

Reply:
Indeed I do agree, in fact initially the code was changed to use indented results in tests, but I was told in another comment that it [wasn't needed](https://github.com/ubuntu/oidc-broker/pull/41#discussion_r1659300492). While imho having the results indented is quite important for helping reviewers and spot changes.